src/lib/react-minimap/components/Child.js

```diff
@@ -75,7 +75,10 @@ export default class Child extends Component {
             displayTitle = shortTitle;
         }
 
-        const className = (this.props.inEditMode && !this.state.visible) ? 'shaded' : '';
+        var className = '';
```

Comment:
As a matter of practice, we try not to use `var` in any code we add toexisting tools. Could we change this to a let?